docs-core/src/main/java/com/sismics/docs/core/service/InboxService.java

```diff
@@ -235,6 +264,33 @@ private void importMessage(Message message) throws Exception {
             FileUtil.createFile(fileContent.getName(), null, fileContent.getFile(), fileContent.getSize(),
                     document.getLanguage(), "admin", document.getId());
         }
+
+        if (ConfigUtil.getConfigBooleanValue(ConfigType.INBOX_DELETE_IMPORTED)) {
+            message.setFlag(Flags.Flag.DELETED, true);
+        }
+    }
+
+    /**
+     * Fetches a HashMap with all tag names in lowercase as keys and their respective ids as values.
+     *
+     * @return HashMap with all tags or null if not enabled
+     */
+    private HashMap<String, String> getAllTags() {
+        if (!ConfigUtil.getConfigBooleanValue(ConfigType.INBOX_AUTOMATIC_TAGS)) {
+            return null;
+        }
+        TagDao tagDao = new TagDao();
+        List<TagDto> tags = tagDao.findByCriteria(new TagCriteria().setTargetIdList(Lists.newArrayList("admin")), new SortCriteria(1, true));
```

Comment:
You can put `setTargetIdList(null)` to fetch all tags ignoring access rights.

Reply:
```suggestion
        List<TagDto> tags = tagDao.findByCriteria(new TagCriteria().setTargetIdList(null), new SortCriteria(1, true));
```